app/views/items/show.html.erb

```diff
@@ -6,22 +6,25 @@
 <h4><%= @headline %></h4>
 <ul id="review_list">
   <% if @reviews.empty? %>
```

Comment:
It would make sense to put the `<ul>` *inside* the `if`... Why create a "review list" ul with a single list item (that's not even a review)? HTML should be more semantic - if a `<ul>` represents a `review_list`, every `<li>` should be a review.

```
<% if @reviews.empty? %>
  <p>No reviews yet!</p>
<% else %>
  <ul>
    <% @reviews.each ... %>
  </ul>
<% end %>
```